File: feedback_pipeline/generations/cpp/files_with_feedback/question-85-pass-1.cpp
#include <bits/stdc++.h>
using namespace std;
void printPair(int n) {
    for (int i = 1; i <= n; i++) {
        for (int j = i; j <= n; j++) {
            if (__gcd(i, j) == n && __lcm(i, j) == n) {
                cout << i << " " << j << endl;
                return;
            }
        }
    }
}
int main() {
    int n;
    cin >> n;
    printPair(n);
    return 0;
}

File: results/fine_tuning/test_output_files/question-85-pass-1.py
def printPair(n):
    for (i in range(1, n + 1))
        for (j in range(1, n + 1))
            if (gcd(i, j) == n and lcm(i, j) == n):
                print(i, j)


if __name__ == "__main__":
    n = 12
    printPair(n)
